lib/mail-archive-helper.ts

```diff
@@ -138,10 +138,9 @@ export class MailArchiveGitHelper {
                     MailArchiveGitHelper.mbox2markdown(parsed);
 
                 if (issueCommentId) {
-                    const result =  await this.githubGlue
-                        .addPRCommentReply(pullRequestURL, issueCommentId,
-                                           comment);
-                    issueCommentId = result.id;
+                    await this.githubGlue.addPRCommentReply(pullRequestURL,
+                                                            issueCommentId,
+                                                            comment);
```

Comment:
This seems to be another ESLint-related change, rather than an Octokit-related one?

Reply:
`result` was removed so I chose this reformat (`await this.githubGlue` looked lonely on a line by itself).

Reply:
👍